ReactiveAnimation/Constraint.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  Constraint.swift
+//  ReactiveAnimation
+//
+//  Created by Indragie on 3/25/15.
+//  Copyright (c) 2015 ReactiveCocoa. All rights reserved.
+//
+
+import Foundation
+import ReactiveCocoa
+
+// Wraps an NSLayoutConstraint for animating its constant using signals.
+public struct RANConstraint {
+	private weak var layoutConstraint: NSLayoutConstraint?
+	private let willDealloc: SignalProducer<(), NoError>
+	
+	// Creates a wrapper for the given layout constraint.
+	public init(_ layoutConstraint: NSLayoutConstraint) {
+		self.layoutConstraint = layoutConstraint
+		self.willDealloc = layoutConstraint.rac_willDeallocSignal().toSignalProducer()
```

Comment:
Ideally there would be a better way to determine whether a constraint becomes inactive than deallocation. `NSLayoutConstraint` has an `active` property, but the docs don't explicitly say its KVO-able.